content/03.results.md

```diff
@@ -49,6 +49,7 @@ This creates a gradient of similarity from genus to superkingdom.
 Path selection using the representative genomes in GTDB rs202 resulted in 4095 paths comprised of 9213 unique genomes (8790 Bacteria, 333 Archaea).
 These paths include genome comparisons across 40 phyla (36 Bacteria, 4 Archaea), covering roughly a quarter of the 169 phyla (149 Bacteria, 20 Archaea) in GTDB release rs202.
 While paths are limited to taxonomies with at least two GTDB representative genomes for each taxonomic rank, these paths provide a rich resource for comparisons at increasing evolutionary distances.
+(CTB: are these available for download?)
```

Comment:
Protein databases are not yet downloadable - we should make them available prior to or simultaneously with preprinting. Are you suggesting I also make an "evolpaths" database with just the genomes selected for this analysis. That's now very doable with sig extract --> zip formats, can do.

As we make protein db's available, we will also need to add documentation about how and when to use protein search (and/or just link this paper)

Reply:
either directly available, or perhaps as picklists! how exciting! 😆 

I think making them available for this paper is first thing, then we can (at same time, or later) write documentation.